frontend/apps/monkvision/components/chart-box/chart-box.mjs

```diff
@@ -100,6 +108,8 @@ async function _refreshData(element, force) {
 			}
 			data.table = {headers, rows};
 		}
+		data.exportCSV = frameworkUtils.parseBoolean(element.getAttribute("exportCSV"));
+		if(data.exportCSV) data.exportCSVIcon = element.getAttribute("exportCSVIcon") || "./components/chart-box/img/download_icon.svg";
```

Comment:
There is always a space between if and the expression ... it should be `if (data.exportCSV)` not `if(data.exportCSV)`

Code MUST look good! I have said it a few times now - variables named correctly, code formatted perfectly - it MUST read like a book!!